frontend/src/layout/navigation/TopBar/Announcement.scss

```diff
@@ -0,0 +1,51 @@
+@import '../../../styles/mixins';
```

Comment:
Would be nice to get a story to see what these changes look like / prevent future regressions

Reply:
We need to add the cloud announcements feature in 3000 and can add the stories there as well